### PR TITLE
feat: add continuous profiling nodejs section

### PR DESCRIPTION
**Pull request by JonasBa**
## DESCRIBE YOUR PR
*Tell us what you're changing and why. If your PR **resolves an issue**, please link it so it closes automatically.*

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [x] Urgent deadline (GA date, etc.): 9/3
- [ ] Other deadline: 
- [ ] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Comment by JonasBa** [comment]
We need an SDK change to go out before this can be merged and the API is exposed (hence the TODO version comment)

File: docs/platforms/javascript/guides/node/profiling/index.mdx [hunk 1]
```diff
@@ -67,6 +67,40 @@ Sentry.startSpan(
 );
 ```
 
+## Enable Continuous Profiling
+
+<Include name="feature-stage-beta.mdx" />
+
+_(New in version @TODO)_
+
+The current profiling implementation stops the profiler automatically after 30 seconds (unless you manually stop it earlier). Naturally, this limitation makes it difficult to get full coverage of your app's execution. We now offer an experimental continuous mode, where profiling data is periodically uploaded while running, with no limit to how long the profiler may run.
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/node/profiling/index.mdx:76]:
```suggestion
The current profiling implementation stops the profiler automatically after 30 seconds (unless you manually stop it earlier). Naturally, this limitation makes it difficult to get full coverage of your app's execution. We now offer an experimental continuous mode, where profiling data is periodically uploaded while running, with no limit on how long the profiler may run.
```

File: docs/platforms/javascript/guides/node/profiling/index.mdx [hunk 2]
```diff
@@ -67,6 +67,40 @@ Sentry.startSpan(
...
+
+To get started with continuous profiling, you can start and stop the profiler directly with `Sentry.profiler.startProfiling` and `Sentry.profiler.stopProfiling`.
+
+<Note>
+
+If you had previously set `profilesSampleRate` or `profilesSampler` to use transaction-based profiling, then remove those lines of code from your configuration.
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/node/profiling/index.mdx:82]:
```suggestion
If you previously set `profilesSampleRate` or `profilesSampler` to use transaction-based profiling, you must remove those lines of code from your configuration in order to use continuous profiling.
```

File: docs/platforms/javascript/guides/node/profiling/index.mdx [hunk 3]
```diff
@@ -67,6 +67,40 @@ Sentry.startSpan(
...
+})
+
+Sentry.profiler.startProfiling();
+
+// run some code here
+
+Sentry.profiler.stopProfiling();
+```
+
+These new APIs do not offer any sampling functionality—every call to start the profiler. If you are interested in reducing the amount of profiles that run, you must take care to do it at the callsites.
```
Comment (vivianyentran) [inline docs/platforms/javascript/guides/node/profiling/index.mdx:100]:
```suggestion
These new APIs do not offer any sampling functionality—every call to start the profiler will run and start sending profiling data. If you are interested in reducing the amount of profiles that run, you must take care to do it at the callsites.
```

**Review by vivianyentran — APPROVED** [review]
Thanks for adding!

**Comment by JonasBa** [comment]
Thanks for reviewing this @vivianyentran!

**Pull request merged**